pkg/authorization/workspace_content_authorizer_test.go

```diff
@@ -47,13 +47,21 @@ func newUser(name string, groups ...string) *user.DefaultInfo {
 	}
 }
 
+func newUserWithExtra(name string, extra map[string][]string, groups ...string) *user.DefaultInfo {
```

Comment:
Can we add a comment on this? What does name mean in this context?

Reply:
not much sense actually to have this func

Reply:
done